src/components/StackPage.js

```diff
@@ -17,12 +23,35 @@ class FlashCardMini extends Component {
 }
 
 class StackPage extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      deleting: false
+    }
+  }
+
   render () {
-    const stack = this.props.stack
+    const { stack, onDeleteStack } = this.props
     return (
       <div className='StackPage'>
+        <Modal isActive={this.state.deleting}>
+          <ModalBackground />
+          <ModalCard>
+            <ModalCardHeader>
+              <ModalCardTitle>Delete {stack.title}?</ModalCardTitle>
+              <Delete onClick={() => this.setState({deleting: false})} />
+            </ModalCardHeader>
+            <ModalCardBody>
+              This will delete your stack FOREVER!!!
```

Comment:
This was meant to be a temporary message, but it's kind of great. Let's leave it.